### PR TITLE
Upgrade to GAP.jl 0.6

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

**Comment by fingolfin** [comment]
@ThomasBreuer @fieker merge?

I am going to write a news item for GAP.jl 0.6 now

**Comment by ThomasBreuer** [comment]
Yes, let us try the new version.

File: experimental/GaloisGrp/GaloisGrp.jl [hunk 1]
```diff
@@ -12,7 +12,8 @@ import Hecke: orbit, fixed_field
 
 
 function __init__()
-  GAP.Packages.load("ferret", install = true)
+  GAP.Packages.install("ferret"; interactive=false, quiet=true)
+  GAP.Packages.load("ferret")
```
Comment (fingolfin) [inline experimental/GaloisGrp/GaloisGrp.jl:16]: @ThomasBreuer I guess we should support `quiet` for `load` as well? Or perhaps the "install" mode in "load" should set `quiet=true` by default?

Comment (ThomasBreuer) [reply experimental/GaloisGrp/GaloisGrp.jl:16]: I am in favor of the second variant: `load` should call `install` with `quiet = true`.

With the above change, any call of `__init__` will install a new version of the package, not only in the situation when the package is not yet installed. Is this intended?

Comment (ThomasBreuer) [reply experimental/GaloisGrp/GaloisGrp.jl:16]: Update of my above comment: As @fingolfin said, `GAP.Packages.install` does not install the package if the newest version is already installed. However, in order to find out what the newest version is, the function fetches the current `PackageInfo.g` file, thus one is in trouble if one is offline.

Comment (fingolfin) [reply experimental/GaloisGrp/GaloisGrp.jl:16]: Sure, so let's improve this in GAP.jl so that we can return to using just `load`

**Pull request merged**